File: seung365/programmers/09.py
def solution(number):
  arr = []
  while(number >=1):
    check = number%2
    number = number//2
    if check == 0:
      arr.append(0)
    else:
      arr.append(1)
  binary = ''
  for i in range(len(arr)):
    binary += str(arr.pop())

  return binary

print(solution(10))
print(solution(27))
print(solution(12345))

File: mezzang/book/09.js
function solution(decimal){
    const stack = [];

    while (decimal > 0){
        const remainder = decimal % 2;
        stack.push(remainder);
        decimal = Math.floor(decimal / 2); // 소수점 이하 버리기
    }

    let binary = "";
    while (stack.length > 0){
        binary += stack.pop();
    }

    return binary;
}
console.log(solution(8));